BTCPayServer.Client/Models/PointOfSaleAppData.cs

```diff
@@ -15,6 +15,51 @@ public class AppDataBase
     
     public class PointOfSaleAppData : AppDataBase
     {
-        // We can add POS specific things here later
+        public string Title { get; set; }
+        public string DefaultView { get; set; }
+        public bool ShowCustomAmount { get; set; }
+        public bool ShowDiscount { get; set; }
+        public bool EnableTips { get; set; }
+        public string Currency { get; set; }
+        public string Template { get; set; }
+        public string ButtonText { get; set; }
+        public string CustomButtonText { get; set; }
+        public string CustomTipText { get; set; }
+        public string CustomCSSLink { get; set; }
+        public string NotificationUrl { get; set; }
+        public string RedirectUrl { get; set; }
+        public string Description { get; set; }
+        public string EmbeddedCSS { get; set; }
+        public bool? RedirectAutomatically { get; set; }
+        public string RequiresRefundEmail { get; set; }
+    }
+
+    public class CrowdfundAppData : AppDataBase
+    {
+        public string Title { get; set; }
+        public bool Enabled { get; set; }
+        public bool EnforceTargetAmount { get; set; }
+        public DateTime? StartDate { get; set; }
+        public string TargetCurrency { get; set; }
+        public string Description { get; set; }
+        public DateTime? EndDate { get; set; }
```

Comment:
Date time offset is preferred.
```suggestion

        [JsonConverter(typeof(NBitcoin.JsonConverters.DateTimeToUnixTimeConverter))]
        public DateTimeOffset? EndDate { get; set; }
```